charts/tekton-apps/templates/wordpress/argocd_applications.yaml

```diff
@@ -185,15 +185,15 @@ spec:
 
         customLivenessProbe:
           httpGet:
-            path: /wp-login.php
+            path: /wp-includes/js/jquery/jquery.min.js
```

Comment:
This is a static file - so we don't get any information if PHP scripts actually work or not.

`install.php` script used to work, doesn't it?

Reply:
technically this file should be removed after the installation
https://www.westhost.com/knowledgebase/article/how-to-avoid-looking-like-a-new-wordpress-install/